openedx/adg/admin_dashboard/admin_settings/views.py

```diff
@@ -298,6 +316,23 @@ def update_permissions(self, forms, invitations):
             user = User.objects.get(email=invitation[EMAIL])
             update_user_permissions(user, [group], is_staff=True)
 
+    def remove_permissions(self, forms, invitations):
+        """
+        Iterate over the invitations dictionary and remove the permissions of all the admins.
+
+        Arguments:
+            forms (list): a list containing InvitationForm objects
+            invitations (dict): dictionary containing invitations
+        """
+        for form, invitation in zip(forms, invitations.values()):
+            if form.instance.pk:
+                form.save()
+
+            user = User.objects.get(email=invitation[EMAIL])
```

Comment:
Can we use this method here?
https://github.com/OmnipreneurshipAcademy/edx-platform/blob/master/openedx/adg/admin_dashboard/admin_settings/helpers.py#L96